catalog/transformer/catalog-transformer-generic/src/main/java/ddf/catalog/transformer/generic/xml/SaxEventHandler.java

```diff
@@ -0,0 +1,28 @@
+package ddf.catalog.transformer.generic.xml;
+
+import java.util.List;
+
+import org.xml.sax.Attributes;
+import org.xml.sax.ContentHandler;
+import org.xml.sax.DTDHandler;
+import org.xml.sax.EntityResolver;
+import org.xml.sax.ErrorHandler;
+
+import ddf.catalog.data.Attribute;
+
+public interface SaxEventHandler extends EntityResolver, DTDHandler, ContentHandler, ErrorHandler {
```

Comment:
Is this api being exposed for implementation? If so, it should be in a separate api bundle and not in this implementation bundle. I'm asking because it's not clear if this interface is here for that purpose or not.